packages/components/package.json

```diff
@@ -32,6 +34,7 @@
     "@chakra-ui/theme-tools": "^1.3.1",
     "@emotion/react": "^11",
     "@emotion/styled": "^11",
+    "@testing-library/jest-dom": "^5.16.1",
```

Comment:
do we really dependency?, and in any case, it should be a `devDependency` no a production one

Reply:
my bad when adding it in the production dependencies, sorry, but about if it is necessary, well, as they say in their documentation: **These will make your tests more declarative, clear to read and to maintain.**
https://github.com/testing-library/jest-dom#the-problem.
but I can perfectly remove it if you and @guilean don't like it. NP